lib/install/config/loaders/core/assets.js

```diff
@@ -1,12 +1,30 @@
-const { env, publicPath } = require('../configuration.js')
+const Url = require('url-parse')
+const { env, paths, publicPath } = require('../configuration.js')
+
+// Make sure host has slashes
+const ensureSlashes = ({ href, slashes }) => {
+  if (slashes) { return `${href}/${paths.entry}/` }
+  return `//${href}/${paths.entry}/`
```

Comment:
This should be a regular if/else conditional, rather than a double exit function. Same with the other methods here.